### PR TITLE
[docs](resource-group) add relevant documents to the resource group

### PR DESCRIPTION
**Pull request by luozenglin**
# Proposed changes

Issue Number: close #xxx

## Problem summary

Describe your changes.

## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
* [ ] Has document been added or modified
* [ ] Does it need to update dependencies
* [ ] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: docs/en/docs/sql-manual/sql-reference/Data-Definition-Statements/Alter/ALTER-RESOURCE-GROUP.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+You must not modify the memory_limit property in such a way that the sum of all memory_limit values exceeds 100%.
+
+### Example
+
+1. Modify the resource group named g1:
+
+    ```sql
+    alter resource group g1
+    properties (
+        "cpu_share"="30",
```
Comment (yiguolei) [inline docs/en/docs/sql-manual/sql-reference/Data-Definition-Statements/Alter/ALTER-RESOURCE-GROUP.md:59]: why cpu share is just a number and memlimit is percent?

Comment (luozenglin) [reply docs/en/docs/sql-manual/sql-reference/Data-Definition-Statements/Alter/ALTER-RESOURCE-GROUP.md:59]: 1. For the convenience of checking the sum of memory_limit on the fe side does not exceed 100%;
2. There is no need to modify the memory_limit after memory expansion and contraction on the cloud using a percentage.
And cpu_share is an existing concept in linux.

File: docs/zh-CN/docs/admin-manual/resource-group.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+创建资源组详细可参考：[CREATE-RESOURCE-GROUP](../sql-manual/sql-reference/Data-Definition-Statements/Create/CREATE-RESOURCE-GROUP.md)，另删除资源组可参考[DROP-RESOURCE-GROUP](../sql-manual/sql-reference/Data-Definition-Statements/Drop/DROP-RESOURCE-GROUP.md)；修改资源组可参考：[ALTER-RESOURCE-GROUP](../sql-manual/sql-reference/Data-Definition-Statements/Alter/ALTER-RESOURCE-GROUP.md)；查看资源组可参考：[RESOURCE_GROUPS()](../sql-manual/sql-functions/table-functions/resource-group.md)和[SHOW-RESOURCE-GROUPS](../sql-manual/sql-reference/Show-Statements/SHOW-RESOURCE-GROUPS.md)。
+
+3. 开启pipeline执行引擎，资源组cpu隔离基于pipeline执行引擎实现，因此需开启session变量：
+```
+set experimental_enable_pipeline_engine = true;
+```
+
+4. 设置session变量 resource_group 指定使用的资源组，默认为默认资源组`normal`:
+```
+resource_group = g1;
```
Comment (wangbo) [inline docs/zh-CN/docs/admin-manual/resource-group.md:67]:
```suggestion
set resource_group = 'g1';
```

File: docs/zh-CN/docs/admin-manual/resource-group.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+);
+```
+创建资源组详细可参考：[CREATE-RESOURCE-GROUP](../sql-manual/sql-reference/Data-Definition-Statements/Create/CREATE-RESOURCE-GROUP.md)，另删除资源组可参考[DROP-RESOURCE-GROUP](../sql-manual/sql-reference/Data-Definition-Statements/Drop/DROP-RESOURCE-GROUP.md)；修改资源组可参考：[ALTER-RESOURCE-GROUP](../sql-manual/sql-reference/Data-Definition-Statements/Alter/ALTER-RESOURCE-GROUP.md)；查看资源组可参考：[RESOURCE_GROUPS()](../sql-manual/sql-functions/table-functions/resource-group.md)和[SHOW-RESOURCE-GROUPS](../sql-manual/sql-reference/Show-Statements/SHOW-RESOURCE-GROUPS.md)。
+
+3. 开启pipeline执行引擎，资源组cpu隔离基于pipeline执行引擎实现，因此需开启session变量：
+```
+set experimental_enable_pipeline_engine = true;
+```
+
+4. 设置session变量 resource_group 指定使用的资源组，默认为默认资源组`normal`:
```
Comment (wangbo) [inline docs/zh-CN/docs/admin-manual/resource-group.md:65]:
```suggestion
4.  查询绑定资源组。目前主要通过指定session变量的方式绑定查询到资源组。如果用户不指定资源组，那么查询默认会提交到`normal`资源组
```

File: docs/zh-CN/docs/admin-manual/resource-group.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+2. 创建资源组：
+```
+create resource group if not exists g1
+properties (
+    "cpu_share"="10",
+    "memory_limit"="30%",
+    "enable_memory_overcommit"="true"
+);
+```
```
Comment (wangbo) [inline docs/zh-CN/docs/admin-manual/resource-group.md:57]:
```suggestion
目前default为系统保留的关键字，因此不建议将资源组命名为default
```
```

File: docs/zh-CN/docs/sql-manual/sql-reference/Data-Definition-Statements/Alter/ALTER-RESOURCE-GROUP.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+```sql
+ALTER RESOURCE GROUP "rg_name"
+PROPERTIES (
+    property_list
+);
+```
+
+注意：
+
+修改 memory_limit 属性时不可使所有 memory_limit 值的总和超过100%。
```
Comment (wangbo) [inline docs/zh-CN/docs/sql-manual/sql-reference/Data-Definition-Statements/Alter/ALTER-RESOURCE-GROUP.md:50]:
```suggestion
修改 memory_limit 属性时不可使所有 memory_limit 值的总和超过100%。
支持修改部分属性，例如只修改cpu_share的话，properties里只填cpu_share即可
```

**Comment by luozenglin** [comment]
run buildall

**Review by luzhijing — APPROVED** [review]
LGTM

**Pull request merged**